rosie/chamber_of_deputies/classifiers/election_expenses_classifier.py

```diff
@@ -14,11 +14,11 @@ class ElectionExpensesClassifier(TransformerMixin):
         Brazilian Federal Revenue category of companies, preceded by its code.
     """
 
-    def fit(self, X):
+    def fit(self, dataset):
```

Comment:
Maybe this is just be being annoying, but I think that conceptually this refactor is here _also_ for these kind of semantic changes too. That said: Pandas jargon uses data frames instead of datasets. What do you think about `dataframe` instead of `dataset`?

Reply:
I got this tip from the test: `self.dataset = pd.read_csv('rosie/chamber_of_deputies/tests/fixtures/election_expenses_classifier.csv',
                                   dtype={'name': np.str, 'legal_entity': np.str})`

So I believe that it should change too. Is it right?

Just a newbie question. What is the difference between a `dataframe` and a `dataset`?

Reply:
> What is the difference between a `dataframe` and a `dataset`?

Dataset is simply a collection os sets (groups) of information. A data frame is more specific, it's a 2 dimensional structure holding data ; )